### PR TITLE
BaSD-MS-99: Admin Home View

### PR DESCRIPTION
**Pull request by ripellizzari**
### **Create Admin Layout acording to the screenflow**

Admin for testing: 
email: ["admin@rr.com"]
password:"12345678"

- [x] Create Admin Header
- [x] Create Admin Side Bar Menu

The Admin Home Page has a display for Desktop.

### CODE REVIEW

File: src/Components/Admin/Admins/admins.module.css [hunk 1]
```diff
@@ -17,6 +17,9 @@
 }
 
 .list {
+  display: flex;
```
Comment (NicoCagnina) [inline src/Components/Admin/Admins/admins.module.css:20]: the list should not have a display flex

File: src/Components/Admin/Admins/admins.module.css [hunk 2]
```diff
@@ -17,6 +17,9 @@
...
+  overflow-x:auto;
```
Comment (NicoCagnina) [inline src/Components/Admin/Admins/admins.module.css:21]: this line is unnecessary

File: src/Components/Admin/Admins/admins.module.css [hunk 3]
```diff
@@ -17,6 +17,9 @@
...
+  flex-direction: column;
```
Comment (NicoCagnina) [inline src/Components/Admin/Admins/admins.module.css:22]: this line is unnecessary

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

File: src/Components/Admin/Admins/admins.module.css [hunk 4]
```diff
@@ -17,6 +17,9 @@
...
   border-collapse: collapse;
   margin: 10px 0;
```
[the remark this answers is not in the document]
Comment (ripellizzari) [reply src/Components/Admin/Admins/admins.module.css:24]: Done!

File: src/Components/Admin/Admins/admins.module.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (ripellizzari) [reply src/Components/Admin/Admins/admins.module.css]: Done!

Comment (ripellizzari) [reply src/Components/Admin/Admins/admins.module.css]: Done!

File: src/Components/Admin/Admins/admins.module.css (hunk 2 above)
Comment (ripellizzari) [reply src/Components/Admin/Admins/admins.module.css]: Done!

File: src/Components/Admin/Admins/admins.module.css (hunk 3 above)
Comment (ripellizzari) [reply src/Components/Admin/Admins/admins.module.css]: Done!

**Review by NicoCagnina — APPROVED** [review]

**Pull request merged**